### ISSUE TITLE
Transfering a user

### ISSUE THREAD

**Issue by darkman088** [qTox/qTox#194]
Hello,

I am new to tox. I first tried uTox, then I saw that there is also qTox and decided to give it a try as well!
Though qTox does not recognize my user. I copied both tox_save and utox_save files and even tried to rename utox_save to qtox_save, but I still cannot see my original ID in qTox.
Please help on that! Thank you!

**Comment by tux3** [comment]
You just need to copy tox_save into <ConfigDirectory>/tox/tox_save

On Windows <ConfigDirectory> should be %APPDATALOCAL%/, on Linux ~/.config/, and on Mac ~/Library/Preferences

**Comment by darkman088** [comment]
Hello,

unfortunately this is not correct :-( There is no %APPDATALOCAL% on Windows, which unfortunately is my platform. There is %APPDATA% which gets resolved to C:\Users<myuser>\AppData\Roaming. I have created a tox subfolder and put the tox_save file there. When I start qTox, it shows me again some new identity.
I have also tried to create a tox directory under C:\Users<myuser>\AppData\Local and I have put the tox_save file there, but it did not work either. Kindly support me on that!

**Comment by dubslow** [comment]
Rename "utox_save" to "data". That is currently the only filename recognized by qTox.

(I have an old branch that was meant to improve this, but I've been incredibly lazy about it for the last number of weeks)

**Comment by darkman088** [comment]
Hi Dubslow and thank you!
Are you able to see my other problems with uTox ? Maybe you know the solution for them in qTox ?

**Comment by darkman088** [comment]
Dear Dubslow, I have renamed the utox_save files to data in all the 3 locations:

C:\Users<myuser>\AppData\Roaming\tox
C:\Users<myuser>\AppData\Local\tox
C:\Users<myuser>\AppData\Local\tox\qTox

Unfortunately it does not work and I am unable to see that other identity and the contacts which I have there... Kindly advise on how to do that...

**Comment by dubslow** [comment]
Remove all folders named "tox" that you have created, as well as any other tox folders in the \AppData\ directory or any of its subfolders.

Then run qTox once more -- it will create the tox folder in whatever location it uses, and in that folder you should delete the "data" file there (the one just created by qTox) and copy your utox_save as "data".

**Comment by darkman088** [comment]
Heeeey, thanks for the hint, I figured it out, though it is quite different than what you said!
So, here is the correct procedure in case someone else would be interested:
So, I did what you suggested, but no folder was created under %APPDATA% and all of its subfolders. Then I realized, that I have enabled the Make Tox portable option! So I started to search for a file, called data under the qTox directory structure and I found it right there in the main directory. I copied utox_save as data, but this didn't solve the problem either. So, the correct approach is to copy tox_save as data and then it finally worked!!!

Many thanks!!!
